### ISSUE TITLE
Chenge /etc/lenovo_fix.conf to /etc/throttled.conf

### ISSUE THREAD

**Issue by ghost** [erpalma/lenovo-throttling-fix#273]
Hi,

When I git pull your repo for updates, I always make those changes:

- rename lenovo_fix.py to throttled.py
- change in Line 899
`parser.add_argument('--config', default='/etc/lenovo_fix.conf', help='override default config file path')`
to
`parser.add_argument('--config', default='/etc/throttled.conf', help='override default config file path')`

It will be great if you make thoses changes in your repo:
- lenovo_fix.py to throttled.py
- /etc/lenovo_fix.conf to /etc/throttled.conf
- change the default config file in line 899 of throttled.py

If that is good for you, then I'll send a request to Debian Teams to add "throttled tool" it in their Stable/Unstable repo

**Comment by erpalma** [comment]
I agree with you that throttled.conf would be a better filename. I'm just afraid about backward compatibility. Do you think it is enough to add a few warnings in the readme or we should upgrade the install script? 

Adding throttled to the Debian repo would be great!

**Comment by jJit0** [comment]
I think you should do both, warn new users about the change, and update the INSTALL script.

I do it manually on each pull. I think "throttled" sounds better than "lenovo_fix"

Yeah it is, once you commit your changes, I'll send an add request to Debian Maintainers.
Your tool is awesome and it should be added into Debian :wink:

**Comment by jJit0** [comment]
Awesome!!!! You did it :heart:

Here is the package request I created: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=998132

If you have any comment to add in the request, just send an email to 998132@bugs.debian.org

**Comment by nerijus** [comment]
I updated to the latest git and ran `sudo ./install.sh`:
```
Updating config filename
mv: cannot stat 'etc/lenovo_fix.conf': No such file or directory
```
I see that `/etc/lenovo_fix.conf` was not renamed to `/etc/throttled.conf`.

**Comment by erpalma** [comment]
Fixed in latest commit, thanks!

**Comment by nerijus** [comment]
I did a `git pull`, `sudo ./install.sh`:
```
Copying config file
Config file already exists, skipping
Copying systemd service file
```
Newer `throttled.py` was not copied to `/opt/throttled`. Had to copy it manually and start service.

**Comment by erpalma** [comment]
That's quite strange:
```
echo "Copying core files"
cp requirements.txt throttled.py mmio.py "$INSTALL_DIR"
```
